src/chrome/content/rules/LightningMaps.org.xml

```diff
@@ -0,0 +1,22 @@
+<!--
+	Invalid certificate:
+		tiles-radar.lightningmaps.org
+		www1.lightningmaps.org
+		www2.lightningmaps.org
+		www3.lightningmaps.org
+
+-->
+<ruleset name="LightningMaps.org">
+
+	<target host="lightningmaps.org" />
+	<target host="www.lightningmaps.org" />
+	<target host="forum.lightningmaps.org" />
+	<target host="images.lightningmaps.org" />
+	<target host="tiles.lightningmaps.org" />
```

Comment:
I get 503 for https://tiles.lightningmaps.org/, can you add a `test` for it? 
There is a working URL: https://tiles.lightningmaps.org/?x=9&y=5&z=4&s=256&t=5&T=6233330